src/cmd/linuxkit/run_aws.go

```diff
@@ -0,0 +1,186 @@
+package main
+
+import (
+	"encoding/base64"
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+)
+
+const (
+	defaultAWSMachine  = "t2.micro"
+	defaultAWSDiskSize = 0
+	defaultAWSDiskType = "gp2"
+	defaultAWSZone     = "a"
+	// Environment variables. Some are non-standard
+	awsMachineVar  = "AWS_MACHINE"   // non-standard
+	awsDiskSizeVar = "AWS_DISK_SIZE" // non-standard
+	awsDiskTypeVar = "AWS_DISK_TYPE" // non-standard
+	awsZoneVar     = "AWS_ZONE"      // non-standard
+)
+
+// Process the run arguments and execute run
+func runAWS(args []string) {
+	flags := flag.NewFlagSet("aws", flag.ExitOnError)
+	invoked := filepath.Base(os.Args[0])
+	flags.Usage = func() {
+		fmt.Printf("USAGE: %s run aws [options] [name]\n\n", invoked)
+		fmt.Printf("'name' is the name of an AWS image that has already been\n")
+		fmt.Printf(" uploaded using 'linuxkit push'\n\n")
+		fmt.Printf("Options:\n\n")
+		flags.PrintDefaults()
+	}
+	machineFlag := flags.String("machine", defaultAWSMachine, "AWS Machine Type")
+	diskSizeFlag := flags.Int("disk-size", 0, "Size of system disk in GiB")
```

Comment:
Should we make this`GB`[ consistent with GCP](https://github.com/linuxkit/linuxkit/blob/master/src/cmd/linuxkit/run_gcp.go#L44)?  Or instead it seems like GCP should get updated to `GiB` since that is more correct...